tests/LocalDriverTest.php

```diff
@@ -9,24 +9,43 @@
 class LocalDriverTest extends TestCase
 {
     protected static LocalDriver $localDriverClass;
-    protected static string $productionFilePath;
+    protected static string $baseFileName;
+    protected static string $baseFile;
     protected static string $tmpDocumentationFilePath;
     protected static array $tmpData;
 
     public function setUp(): void
     {
         parent::setUp();
 
-        self::$productionFilePath ??= __DIR__ . '/../storage/documentation.json';
-        self::$tmpDocumentationFilePath ??= __DIR__ . '/../storage/temp_documentation.json';
+        $documentationDirectory = config('auto-doc.drivers.local.directory');
+        if (!str_ends_with($documentationDirectory, DIRECTORY_SEPARATOR)) {
+            $documentationDirectory .= DIRECTORY_SEPARATOR;
+        }
+
+        self::$baseFileName ??= 'documentation';
+        self::$baseFile ??= storage_path($documentationDirectory . self::$baseFileName . '.json');
+        self::$tmpDocumentationFilePath ??= storage_path('temp_documentation.json');
 
         self::$tmpData ??= $this->getJsonFixture('tmp_data');
 
-        config(['auto-doc.drivers.local.production_path' => self::$productionFilePath]);
+        config(['auto-doc.drivers.local.base_file_name' => self::$baseFileName]);
 
         self::$localDriverClass ??= new LocalDriver();
     }
 
+    public function testDirectoryEndsWithDirectorySeparator()
+    {
+        config(['auto-doc.drivers.local.directory' => config('auto-doc.drivers.local.directory').DIRECTORY_SEPARATOR]);
+        config(['auto-doc.drivers.local.base_file_name' => self::$baseFileName]);
```

Comment:
you've already set this config in constructor, any reasons to make it again in the first test?